### PR TITLE
ADD querystring flags to force h264 and dtx

### PR DESCRIPTION
**Pull request by ggarber**

### CODE REVIEW

File: views/room.ejs [hunk 1]
```diff
@@ -12,15 +12,16 @@
         <title>OpenTok Meet : <%=room%></title>
         <link href='//fonts.googleapis.com/css?family=Ropa+Sans' rel='stylesheet' type='text/css'>
         <link rel="stylesheet" href="/css/main.css" media="screen" title="no title" charset="utf-8">
+        <script src="https://d3brof7fu9hlhe.cloudfront.net/static/rtcstats.min.js" type="text/javascript" charset="utf-8"></script>
+        <script src="/js/commons.min.js" type="text/javascript" charset="utf-8"></script>
+        <script src="/js/webrtc-intercept.bundle.min.js" type="text/javascript" charset="utf-8"></script>
```
Comment (ggarber) [inline views/room.ejs]: I have to intercept PeerConnection before loading opentok.js

File: views/room.ejs [hunk 2]
```diff
@@ -12,15 +12,16 @@
...
         <script type="text/javascript">
           window.OTProperties = { enableErrorReporting: true };
         </script>
         <script src="https://tbdev.tokbox.com/v2/js/opentok.js"></script>
-        <script src="https://d3brof7fu9hlhe.cloudfront.net/static/rtcstats.min.js" type="text/javascript" charset="utf-8"></script>
-
+        <script src="/js/ot-intercept.bundle.min.js" type="text/javascript" charset="utf-8"></script>
```
Comment (ggarber) [inline views/room.ejs]: I have to intercept Raptor.Message.connection.create after loading opentok.js

**Comment by ggarber** [comment]
retest this please

**Comment by aullman** [comment]
@ggarber is there any kind of test we can/should add for this that can verify that H.264 is actually being used? Perhaps something that calls getStats and verifies the codec? We could also add the codec to the stats drop down perhaps? That would make the test easier to write.

**Pull request closed without merging**